assets/roks-metrics/roks-metrics-deployment.yaml

```diff
@@ -32,7 +32,7 @@ spec:
 {{- if .ROKSMetricsSecurityContext }}
 {{- $securityContext := .ROKSMetricsSecurityContext }}
         securityContext:
-          runAsUser: {{ $securityContext.RunAsUser }}
+          runAsNonRoot: {{ $securityContext.RunAsNonRoot }}
```

Comment:
ROKSMetricsSecurityContext is used in other files.  Those should be changed as well.